db/create_tables.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE IF NOT EXISTS `audit` (
+CREATE TABLE IF NOT EXISTS `audit_records` (
```

Comment:
did this file exist in master before this? I thought this would show up as a new file.